### PR TITLE
Alex/updatetolangpackrc1 2 16 2022

### PR DESCRIPTION
**Pull request by smartguest**
Adds in the most recent langpacks for the February release to the RC branch.

Vsixes can be found here: https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=140824&view=artifacts&pathAsName=false&type=publishedArtifacts

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -1779,14 +1779,14 @@
 					},
 					"versions": [
 						{
-							"version": "1.34.0",
-							"lastUpdated": "12/14/2021",
+							"version": "1.35.0",
+							"lastUpdated": "2/23/2022",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:1783]: Use today's date for these - the lastUpdated should be the date the gallery was actually updated.

File: extensionsGallery.json [hunk 2]
```diff
@@ -2364,14 +2364,14 @@
 					},
 					"versions": [
 						{
-							"version": "1.34.0",
-							"lastUpdated": "12/14/2021",
+							"version": "1.35.0",
+							"lastUpdated": "2/16/2022",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:2368]: 2/15/2022

**Review by erpett — APPROVED** [review]

**Comment by smartguest** [comment]
@erpett I forgot to change the date, need to fix it before merging in

**Comment by erpett** [comment]
Updated the test Gallery

**Pull request merged**